doc/services/ipc/ipc_service/backends/ipc_service_icbmsg.rst

```diff
@@ -81,3 +81,299 @@ Samples
 =======
 
 * :ref:`ipc_multi_endpoint_sample`
+
+Detailed Protocol Specification
+===============================
+
+Shared Memory Organization
+--------------------------
+
+The ICBMsg uses two shared memory regions, ``rx-region`` for message receiving, and ``tx-region`` for message transmission.
+The regions do not have to be next to each other nor placed in any specification order nor have the same size.
```

Comment:
```suggestion
The regions do not have to be next to each other nor placed in any specific order nor have the same size.
```

Reply:
I would add a remark that these areas must be aligned to cache boundary and do not overlap.
...
I went further and see there is a remark regarding cache so  maybe this is not needed.